web/src/PickNeighbourhoodMode.svelte

```diff
@@ -159,20 +163,30 @@
   </div>
 
   <div slot="sidebar">
-    <div><Link on:click={newBoundary}>Draw a new boundary</Link></div>
-    <div>
-      <Link on:click={() => ($mode = { mode: "auto-boundaries" })}>
-        Use an auto-generated boundary
-      </Link>
-    </div>
-
-    <ul>
-      {#each boundaryNames as name}
-        <li>
+    <h3 style="padding: 4px 8px; margin-bottom: 0">Neighbourhoods</h3>
+    <ul class="neighbourhood-list">
+      {#each neighbourhoods.features as { properties: { name, area_km2, simd } }}
+        <!-- 
```

Comment:
Hmm. I think `JoinedData` can also work with GJ, by leaving this unset.

```
  export let sourceLayer: string | undefined = undefined;
```

But I haven't played with it here. The other brute force alternative is to make a second <GeoJSON> with zero or one features, and set it here. That's messier, requires copying the feature over.

I'll have a go at this later.